pages/qdm-to-qicore.md

```diff
@@ -1862,27 +1802,17 @@ To capture specific types of "Physical Exam, Recommended" as QICoreObservationNo
 
 ### 8.20 Procedure
 
-QDM defines Intervention and Procedure as:
-* *Intervention* is a course of action intended to achieve a result in
-    the care of persons with health problems that does not involve
-    direct physical contact with a patient. Examples include patient
-    education and therapeutic communication.
-
-* *Procedure* is an act whose immediate and primary outcome
-    (post-condition) is the alteration of the physical condition of the
-    subject. A *procedure* may be a surgery or other type of physical
-    manipulation of a person’s body in whole or in part for purposes of
-    making observations and diagnoses or providing treatment.
+QDM defines Procedure an act whose immediate and primary outcome
+(post-condition) is the alteration of the physical condition of the subject. A
+*procedure* may be a surgery or other type of physical manipulation of a
+person’s body in whole or in part for purposes of making observations and
+diagnoses or providing treatment.
 
 #### 8.20.1 Procedure Vs Intervention
 
-FHIR references both of these concepts using the *Procedure* resource,
-specifically noting a process that involves verification of the
-patient's comprehension or to change the patient's mental state would be
-a Procedure. Therefore, both QDM *datatypes*, Procedure and Intervention
-are included in this section of the QDM to QI-Core mapping especially
-since all of the QDM attributes for each of these QDM *datatypes* are
-identical.
+FHIR references both of this concept using the Procedure resource, specifically
```

Comment:
Suggest changing to "FHIR references both of these concepts using ..."

Reply:
changed as suggested